src/core/public/saved_objects/saved_objects_client.ts

```diff
@@ -183,6 +185,11 @@ const getObjectsToFetch = (queue: BatchQueueEntry[]): ObjectTypeAndId[] => {
 export class SavedObjectsClient {
   private http: HttpSetup;
   private batchQueue: BatchQueueEntry[];
+  /**
+   * if currentWorkspaceId is undefined, it means
+   * we should not carry out workspace info when doing any operation.
```

Comment:
I feel this sounds a bit inaccurate, event `currentWorkspaceId` is not set, the `workspaces` in `options` will be used.

Reply:
You are right, adjust the comment.